coinstacks/ethereum/api/src/index.ts

```diff
@@ -13,6 +13,12 @@ app.use(json())
 app.use(urlencoded({ extended: true }))
 app.use(cors())
 
+app.use('*', (_, res, next) => {
+  res.setHeader('Access-Control-Allow-Origin', '*')
+  res.setHeader('Access-Control-Allow-Methods', 'OPTIONS, GET, POST')
+  next()
+})
+
```

Comment:
Again, `cors()` should be doing this very thing. If that isn't the case, would like to better understand what config we need to pass into the cors function.
```suggestion
```